mmv1/third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -96,6 +96,15 @@ func TestAccBigtableInstance_cluster(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"deletion_protection", "instance_type"}, // we don't read instance type back
 			},
+			{
+				Config: testAccBigtableInstance_clusterModifiedAgain(instanceName, 5),
```

Comment:
I have not been able to figure out how to write a correct test case to reproduce this bug...